layouts/partials/page-meta-links.html

```diff
@@ -1,5 +1,5 @@
 {{ if .File }}
-{{ $pathFormatted := replace .File.Path "\\" "/" -}}
+{{ $pathFormatted := strings.TrimPrefix hugo.WorkingDir $.File.Filename -}}
```

Comment:
The line your are changing was added in support of builds under Windows:

- #163

Removing the line will introduce a regression IMHO.

Reply:
Actually, our tests are passing under windows, so I'll assume that this is still ok. If not, we can address the regression later.